side-navigation.yaml

```diff
@@ -1,194 +1,207 @@
 - heading: Welcome
   subheadings:
-    - title: Get started
-      url: /docs
-    - title: Building with Vanilla
-      url: /docs/building-vanilla
-    - title: Customising Vanilla
-      url: /docs/customising-vanilla
-    - title: What's new in {version}
-      url: /docs/whats-new
-    - title: Migrating to v4
-      url: /docs/migration-guide-to-v4
+    items:
+      - title: Get started
+        url: /docs
+      - title: Building with Vanilla
+        url: /docs/building-vanilla
+      - title: Customising Vanilla
+        url: /docs/customising-vanilla
+      - title: What's new in {version}
+        url: /docs/whats-new
+      - title: Migrating to v4
+        url: /docs/migration-guide-to-v4
 - heading: Base elements
   subheadings:
-    - title: Code
-      url: /docs/base/code
-    - title: Forms
-      url: /docs/base/forms
-    - title: Reset
-      url: /docs/base/reset
-    - title: Separators
-      url: /docs/base/separators
-    - title: Tables
-      url: /docs/base/tables
-    - title: Typography
-      url: /docs/base/typography
-    - title: Paper background
-      url: /docs/base/paper
+    ordering: alphabetical
```

Comment:
Would it work if we move `ordering` up one level (alongside `heading` and `subheadings`, to avoid additional nesting level of `items`)?